From 2121452cb6fe8ffef101426fbddd34f76942ea97 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Wed, 10 Apr 2024 11:30:10 +0300
Subject: [PATCH] [FSW-402] fs1: bootrom_init: build issue fix

with this patch, fixed the build issue that was
introduced when we started using common macro
declarations.

Change-Id: I5e9a1c63c99f1d235b9fb5c6cf98d840af2aeb47
---

diff --git a/arch/fs1/arc/common.c b/arch/fs1/arc/common.c
index 38a4cc0..a65fc4d 100644
--- a/arch/fs1/arc/common.c
+++ b/arch/fs1/arc/common.c
@@ -134,5 +134,5 @@
 	}
 
 	cold_rst_data.arc1_iccm_init_done = 1;
-	regmap_writel(cold_rst_data.data, IO0_HALF0_PSOC_BOOT(mmPSOC_BOOT_CONF_COLD_RST_REG_2));
+	regmap_writel(cold_rst_data.data, IO0_H0_PSOC_BOOT(mmPSOC_BOOT_CONF_COLD_RST_REG_2));
 }
